### PR TITLE
Add Bugdom 1.3.1

### PR DESCRIPTION
**Pull request by ivanholmes**
Based on #6891

### CODE REVIEW

**Comment by ivanholmes** [comment]
Ready to merge from my POV.

File: games-action/bugdom/bugdom-1.3.1.recipe [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+CHECKSUM_SHA256="64f266f7f749d9346b1202fdd191a0e445f6bb90372e32c7250b4fb679150523"
+SOURCE_DIR="Bugdom-$portVersion"
+PATCHES="bugdom-$portVersion.patchset"
+ADDITIONAL_FILES="bugdom.rdef.in"
+srcGitRev2="b9ddab06cdc0b9e4a8085e50974213b6c9625436"
+SOURCE_URI_2="https://github.com/jorio/Pomme/archive/$srcGitRev2.tar.gz"
+CHECKSUM_SHA256_2="96be755cbbcc2f4a3677dd4388599bad4b5cc771a55d927bb8ebe646d7142b52"
+SOURCE_FILENAME_2="Pomme-$srcGitRev2.tar.gz"
+SOURCE_DIR_2="Pomme-$srcGitRev2"
+PATCHES_2="bugdom-$portVersion-source2.patchset"
```
Comment (korli) [inline games-action/bugdom/bugdom-1.3.1.recipe:25]:
```suggestion
SOURCE_URI="https://github.com/jorio/Bugdom/archive/refs/tags/$portVersion.tar.gz"
CHECKSUM_SHA256="64f266f7f749d9346b1202fdd191a0e445f6bb90372e32c7250b4fb679150523"
SOURCE_DIR="Bugdom-$portVersion"
PATCHES="bugdom-$portVersion.patchset"
srcGitRev2="b9ddab06cdc0b9e4a8085e50974213b6c9625436"
SOURCE_URI_2="https://github.com/jorio/Pomme/archive/$srcGitRev2.tar.gz"
CHECKSUM_SHA256_2="96be755cbbcc2f4a3677dd4388599bad4b5cc771a55d927bb8ebe646d7142b52"
SOURCE_FILENAME_2="Pomme-$srcGitRev2.tar.gz"
SOURCE_DIR_2="Pomme-$srcGitRev2"
PATCHES_2="bugdom-$portVersion-source2.patchset"
ADDITIONAL_FILES="bugdom.rdef.in"
```

File: games-action/bugdom/bugdom-1.3.1.recipe [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	bugdom$secondaryArchSuffix = $portVersion
+	app:Bugdom = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
```
Comment (korli) [inline games-action/bugdom/bugdom-1.3.1.recipe:37]:
```suggestion
	lib:libgl$secondaryArchSuffix
	lib:libsdl2_2.0$secondaryArchSuffix
```

File: games-action/bugdom/bugdom-1.3.1.recipe [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsdl2_2.0$secondaryArchSuffix
+	devel:libgl$secondaryArchSuffix
```
Comment (korli) [inline games-action/bugdom/bugdom-1.3.1.recipe:43]:
```suggestion
	devel:libgl$secondaryArchSuffix
	devel:libsdl2_2.0$secondaryArchSuffix
```

File: games-action/bugdom/bugdom-1.3.1.recipe [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
```
Comment (korli) [inline games-action/bugdom/bugdom-1.3.1.recipe:48]:
```suggestion
	cmd:gcc$secondaryArchSuffix
	cmd:make
```

File: games-action/bugdom/additional-files/bugdom.rdef.in [hunk 5]
```diff
@@ -0,0 +1,69 @@
+resource app_flags B_MULTIPLE_LAUNCH;
+
+resource app_version {
+	major  = @MAJOR@,
+	middle = @MIDDLE@,
+	minor  = 0,
```
Comment (augiedoggie) [inline games-action/bugdom/additional-files/bugdom.rdef.in:6]: The template rdef.in file you started with doesn't seem to be using the MINOR part of the version.

File: games-action/bugdom/additional-files/bugdom.rdef.in [hunk 6]
```diff
@@ -0,0 +1,69 @@
+resource app_flags B_MULTIPLE_LAUNCH;
```
Comment (augiedoggie) [inline games-action/bugdom/additional-files/bugdom.rdef.in:1]:
```suggestion
resource app_flags B_EXCLUSIVE_LAUNCH | B_ARGV_ONLY;
```

**Review by augiedoggie — CHANGES_REQUESTED** [review]

File: games-action/bugdom/bugdom-1.3.1.recipe [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+	mkdir -p $appsDir/Bugdom
+	cp -ra Data $appsDir/Bugdom
+	cp -a build/Bugdom $appsDir/Bugdom
+	# put docs in right place
+	mkdir -p $docDir
+	cp -a docs/* $docDir
+
+	local APP_SIGNATURE="application/x-vnd.bugdom"
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
```
Comment (augiedoggie) [inline games-action/bugdom/bugdom-1.3.1.recipe:74]:
```suggestion
	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
	local MINOR="`echo "$portVersion" | cut -d. -f3`"
```

File: games-action/bugdom/bugdom-1.3.1.recipe [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+	local LONG_INFO="$SUMMARY"
+	sed \
+		-e "s|@APP_SIGNATURE@|$APP_SIGNATURE|" \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
```
Comment (augiedoggie) [inline games-action/bugdom/bugdom-1.3.1.recipe:79]:
```suggestion
		-e "s|@MIDDLE@|$MIDDLE|" \
		-e "s|@MINOR@|$MINOR|" \
```

**Review by augiedoggie — CHANGES_REQUESTED** [review]

File: games-action/bugdom/bugdom-1.3.1.recipe (hunk 7 above)
Comment (ivanholmes) [reply games-action/bugdom/bugdom-1.3.1.recipe:74]: Oops, will fix for the others too

**Pull request merged**